doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,353 @@
+# <a name="top"></a>List of HTTP Headers used by Orion
+
+1)  [Accept](#1-accept)        
+2)  [Content-Length](#2-content-length)         
+3)  [Content-Type](#3-content-type)
+4)  [Connection](#4-connection)                
+5)  [Expect](#5-expect)                                                                                
+6)  [Host](#6-host)                        	                                                      
+7)  [Origin](#7-origin)                                                       
+8)  [User-Agent](#8-user-agent)              
+9)  [X-Forwarded-For](#9-x-forwarded-for)                                                               
+10) [X-Real-IP](#10-x-real-ip)
+
+11) [Access-Control-Allow-Origin](#11--access-control-allow-origin)
+12) [Access-Control-Allow-Headers](#12-access-control-allow-headers)                                                                   
+13) [Access-Control-Allow-Methods](#13-access-control-allow-methods)
+14) [Access-Control-Max-Age](#14-access-control-max-age)                                 
+15) [Access-Control-Expose-Headers](#15-access-control-expose-headers)
+16) [Allow](#16-allow)                                                                                    
+17) [Location](#17-location)                                                 
+18) [Ngsiv2-AttrsFormat](#18-ngsiv2-attrsformat)                                    
+19) [Fiware-Service](#19-fiware-service)
+20) [Fiware-Servicepath](#20-fiware-servicepath)
+21) [Fiware-Total-Count](#21-fiware-total-count)
+22) [Fiware-Correlator](#22-fiware-correlator)                                     
+
+23) [X-Auth-Token](#23-x-auth-token)
+
+
+
+# HTTP Headers used by Orion with their Description:
+
+
+
+## 1. Accept
+
+Accept HTTP Header is used in requests. 
+Accept HTTP Header is used with the request URL for accepting the response in well defined JSON format. 
+For example in request URL if we use:
+	
+    curl ... 'Accept: application/json'
+		 
+On this request we'll get the response data in well defined JSON format.
+
+The API response payloads in this specification are based on `application/json` and  (for attribute value 
+type operation) `text/plain` MIME types. Clients issuing HTTP requests with accept types different 
+than those will get a `406 Not Acceptable` error.
+
+[Top](#top)
+
+## 2. Content-Length
+
+This header is used by both requests and responses.
+Content-length HTTP header is a mandatory header in Orion responses.
+it defines the length of the request and response body in octets (8-bit bytes).
```

Comment:
```suggestion
It defines the length of the request and response body in bytes
```

(Explaining that a byte is a 8-bit number is excessive ;) )

Reply:
Fixed in 12b17b7